packages/builder/src/pages/builder/app/[application]/_layout.svelte

```diff
@@ -102,6 +107,15 @@
     }
   }
 
+  const showFreeTrialModal = (planType, freeTrialModal) => {
```

Comment:
To encapsulate logic, this should be part of the modal itself, or to have a wrapper around it. We should have the actual `_layout` as lean as possible

Reply:
Refactored